Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3219,6 +3250,14 @@ public Activity getActivity() {
             return preferredOptions_;
         }
 
+        public List<String> getExcludedFromShareSheet() {
```

Comment:
Can we package lock the get methods for the  include/ exclude arrays since they are not supposed to be used by the dev